src/IoC/Autofac/NuGet/Content/IocBootstrapper.cs.pp

```diff
@@ -1,19 +0,0 @@
-﻿using Kekiri.IoC.Autofac;
-
-namespace $rootnamespace$
```

Comment:
Per our offline discussion, I agree that this bootstrapper is lower value than the cost dealing with emerging tooling.  Maintaining bootstrappers require effort and are fiddly anyway, so they need to add substantial value.  This one does not.